examples/src/main/go/wfa/examples/streaming/hello_streaming_client.go

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2021 The Cross-Media Measurement Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"google.golang.org/grpc"
+	"log"
+	"time"
+	pb "wfa/examples/streaming/grpc"
+)
+
+func main() {
+	serverPort := "50051"
+	address := fmt.Sprintf("localhost:%s", serverPort)
```

Comment:
Any reason for building this string instead of just writing `address := "localhost:50051"`?

Reply:
+1 that.
And you might just want to make this configurable using `flag.String()` and let a user pass the whole "host:port" string, defaulting to `"localhost:50051"`